# HG changeset patch
# Parent 1ce39e639d3110700d7fd7220cc8f80bf33fc0ba
Build: Prune system integration stuff

We already have hotplug, init, and install scripts as a part of
our build process.

Basically, we keep the xen-watchdog script.

diff -r 1ce39e639d31 tools/hotplug/Linux/Makefile
--- a/tools/hotplug/Linux/Makefile
+++ b/tools/hotplug/Linux/Makefile
@@ -60,18 +60,13 @@ all:
 build:
 
 .PHONY: install
-install: all install-initd install-scripts $(HOTPLUGS)
+install: install-initd
 
 # See docs/misc/distro_mapping.txt for INITD_DIR location
 .PHONY: install-initd
 install-initd:
 	[ -d $(DESTDIR)$(INITD_DIR) ] || $(INSTALL_DIR) $(DESTDIR)$(INITD_DIR)
 	[ -d $(DESTDIR)$(SYSCONFIG_DIR) ] || $(INSTALL_DIR) $(DESTDIR)$(SYSCONFIG_DIR)
-	$(INSTALL_PROG) $(XEND_INITD) $(DESTDIR)$(INITD_DIR)
-	$(INSTALL_PROG) $(XENDOMAINS_INITD) $(DESTDIR)$(INITD_DIR)
-	$(INSTALL_PROG) $(XENDOMAINS_SYSCONFIG) $(DESTDIR)$(SYSCONFIG_DIR)/xendomains
-	$(INSTALL_PROG) $(XENCOMMONS_INITD) $(DESTDIR)$(INITD_DIR)
-	$(INSTALL_PROG) $(XENCOMMONS_SYSCONFIG) $(DESTDIR)$(SYSCONFIG_DIR)/xencommons
 	$(INSTALL_PROG) init.d/xen-watchdog $(DESTDIR)$(INITD_DIR)
 
 .PHONY: install-scripts
diff -r 1ce39e639d31 tools/hotplug/common/Makefile
--- a/tools/hotplug/common/Makefile
+++ b/tools/hotplug/common/Makefile
@@ -6,7 +6,7 @@ HOTPLUGPATH="hotplugpath.sh"
 # OS-independent hotplug scripts go in this directory
 
 # Xen scripts to go there.
-XEN_SCRIPTS = $(HOTPLUGPATH)
+XEN_SCRIPTS = 
 XEN_SCRIPT_DATA =
 
 genpath-target = $(call buildmakevars2file,$(HOTPLUGPATH))
